src/sentry/event_manager.py

```diff
@@ -1880,6 +1880,101 @@ def _calculate_span_grouping(jobs, projects):
             sentry_sdk.capture_exception()
 
 
+@metrics.wraps("save_event.get_or_create_performance_group")
+def _get_or_create_performance_group(jobs, projects):
+    # TODO: batch operations (like rate limiting) so we don't repeat for each job
+    for job in jobs:
+        # FEATURE FLAG RANDOMIZER
+        # check options to see if we should be creating performance issues for this org
+        rate = options.get("incidents-performance.rollout-rate")
+        if (rate and rate > random.random()) or True:
+
+            # TODO: get fingerprint
+            existing_grouphash = job.get("fingerprint")
```

Comment:
I like this. We can probably just insert the fingerprint into the job either as part of detection, or another step. I think a job will be able to have multiple fingerprints - even though we're only going to create one issue at the moment, we probably want to treat this as a list so that it just works when we add support for multiple issues per transaction